amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseManagementConfig.java

```diff
@@ -220,6 +220,15 @@ public LeaseManagementConfig metricsFactory(final MetricsFactory metricsFactory)
         }
     };
 
+    private ShardSyncer shardSyncer;
```

Comment:
No need to create the getter

```java
private ShardSyncer shardSyncer = new ShardSyncer();
```

This should work.

Reply:
yes you are right